### PR TITLE
Push raw to pastebin if too long for Discord

### LINKED ISSUE #1037: raw command doesn't trim lines that are too long for the paginator
Sentry Issue: [BOT-6H](https://sentry.io/organizations/python-discord/issues/1757973940/?referrer=github_integration)

The command uses discord.py's default `Paginator`. Can that be replaced with our `LinePaginator`, which automatically moves long text to a new page?

```
RuntimeError: Line exceeds maximum page size 1992
  File "discord/ext/commands/core.py", line 83, in wrapped
    ret = await coro(*args, **kwargs)
  File "bot/cogs/information.py", line 412, in raw
    add_content(title, transformer(item))
  File "bot/cogs/information.py", line 396, in add_content
    paginator.add_line(content.replace('```', '`` `'))
  File "discord/ext/commands/help.py", line 127, in add_line
    raise RuntimeError('Line exceeds maximum page size %s' % (max_page_size))

Error executing command invoked by <REDACTED>: !raw <REDACTED>
```

### PR DESCRIPTION
**Pull request by ChrisLovering**
This also removes the use of a paginator entirely.

Closes #1037

# Examples
## Short messages
Before:
image
After:
image
## Long messages
Before:
Error, see sentry issue linked above
After:
image
Which links to https://paste.pythondiscord.com/XK7WHB7I4I4HDAF36XMY5ZSNEY

### CODE REVIEW

File: bot/exts/info/information.py [hunk 1]
```diff
@@ -525,8 +526,25 @@ def add_content(title: str, content: str) -> None:
...
                 add_content(title, transformer(item))
 
-        for page in paginator.pages:
-            await ctx.send(page, allowed_mentions=AllowedMentions.none())
+        output = "\n".join(lines)
+        if len(output) < 2000-8:  # To cover the backticks and newlines added below.
+            await ctx.send(f"```\n{output}\n```", allowed_mentions=AllowedMentions.none())
+            return
+
+        file = PasteFile(content=output, lexer="python")
```
[the remark this answers is not in the document]
Comment (ChrisLovering) [reply bot/exts/info/information.py]: Good point, done in the [forcepush](https://github.com/python-discord/bot/compare/10032e6d07bb79e11d688814c9b8b562e65f6c03..56ecf7d4177349249f528b5a2b5e86c04cf02e6c)

File: bot/exts/info/information.py [hunk 2]
```diff
@@ -501,14 +503,13 @@ async def send_raw_content(self, ctx: Context, message: Message, json: bool = Fa
...
         raw_data = await ctx.bot.http.get_message(message.channel.id, message.id)
 
-        paginator = Paginator()
+        lines = []
 
         def add_content(title: str, content: str) -> None:
-            paginator.add_line(f"== {title} ==\n")
+            lines.append(f"== {title} ==\n")
             # Replace backticks as it breaks out of code blocks.
             # An invisible character seemed to be the most reasonable solution. We hope it's not close to 2000.
```
[the remark this answers is not in the document]
Comment (ChrisLovering) [reply bot/exts/info/information.py]: Good point, done in the [forcepush](https://github.com/python-discord/bot/compare/10032e6d07bb79e11d688814c9b8b562e65f6c03..56ecf7d4177349249f528b5a2b5e86c04cf02e6c)

I didn't fancy refactoring at this point to only add invisible chars if sending to discord.

Second forcepush is just a rebase onto main.

**Review by wookie184 — APPROVED** [review]
Great stuff! Thanks!

**Review by Xithrius — APPROVED** [review]
long

**Pull request merged**